msteams-platform/tabs/what-are-custom-tabs.md

```diff
@@ -6,40 +6,48 @@ ms.topic: overview
 ms.author: v-laujan
 ms.topic: overview
 ---
-# What are custom tabs in Microsoft Teams?
+# What are Microsoft Teams custom tabs?
 
-Tabs are Teams-aware web experiences embedded in Microsoft Teams. They can be added as part of a channel inside a Team, a group chat, or part of a [personal app](~/foo.md) for an individual user. As part of your app, you can build custom tabs that embed your own web content in Teams, and add Teams-specific functionality using our Teams client SDK.
+Custom tabs are your hosted Teams-aware IFrame webpages embedded in Microsoft Teams. The dedicated tab experience can be added as part of a [channel or group chat](~/foo.md) inside a Team or as a [personal app](~/foo.md) for an individual user. As part of your app, you can add custom tabs, using our [Teams JavaScript client SDK](/javascript/api/overview/msteams-client), to embed your own web content in Teams and add Teams-specific functionality.
```

Comment:
I like starting with defining what a tab is, then transitioning to define what a custom tab is within the context of your team. This creates a nice distinction between a generic tab, and a custom tab.